modules/database/src/main/java/org/eclipse/xpanse/modules/database/service/DeployResourceEntity.java

```diff
@@ -30,14 +30,26 @@ public class DeployResourceEntity {
     @Id
```

Comment:
can we remove this JPA annotation here?

Reply:
yes, we can remove this.
I shouldn't follow the IDE's suggestion with "xxx should have primary key ".